startup/30-detectors_and_cameras_common.py

```diff
@@ -40,10 +38,79 @@
 from ophyd.device import DeviceStatus
 
 
+class XPDCamBase(CamBase):
+    wait_for_plugins = Component(
+        EpicsSignal, "WaitForPlugins", string=True, kind="config"
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.stage_sigs["wait_for_plugins"] = "Yes"
+
+    def ensure_nonblocking(self):
+        self.stage_sigs["wait_for_plugins"] = "Yes"
+        for c in self.parent.component_names:
+            cpt = getattr(self.parent, c)
+            if cpt is self:
+                continue
+            if hasattr(cpt, "ensure_nonblocking"):
+                #                 print(f'cpt: {cpt.name}')
+                cpt.ensure_nonblocking()
```

Comment:
We can reuse the existing class from [`nslsii`](https://github.com/NSLS-II/nslsii/blob/618ca2bdf3a012743f33eb114c97dc097f88a879/nslsii/ad33.py#L37-L52).